### DISCUSSION TITLE
Blocked commands in Combat, also whitelist option

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#4399] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, could you add this section in config also as whitelisted one? For ex. something like: BlacklistWorkLikeWhitelist: true, or something like this.
```
    BlockCommands: true
    AllowedCommands:
    - msg
    - r
    - tell
```

### Basic explanation as to why this feature is needed and/or useful to have: ###
Will be much easier, when I wanna for ex. disable /fly again, because even when fly is disabled, players can use /fly again to enable for them... But I don't wanna block them all commands, because you don't know, what important commands may they need while they're in battle, so I wanna blacklist just fly etc.

**Comment by Teals53** [comment]
add this pls
